python/core/model_processing/onnx_custom_ops/__init__.py

```
@@ -1,16 +1,17 @@
-package_dir = os.path.dirname(__file__)
+package_dir = Path(__file__).parent.as_posix()
 
-__all__ = []
+
+__all__: list[str] = []
 
 for _, module_name, is_pkg in pkgutil.iter_modules([package_dir]):
     if not is_pkg and (module_name != "custom_helpers"):
         importlib.import_module(f"{package_name}.{module_name}")
-        __all__.append(module_name)
+        __all__.append(module_name)  # noqa: PYI056
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `PYI056`)

Remove unused `noqa` directive

(RUF100)